README.md

```diff
@@ -1,12 +1,12 @@
 ![dogescript](doge.gif)
 
-## dogescript 
+## dogescript
```

Comment:
Not sure exactly what this change is, but it doesn't seem relevant to this PR

Reply:
might be CRLF -> LF. I'll check with a .gitattributes :)

Reply:
Ah, it seems there was a ` ` character in there (not a line char)

```
## dogescript ]
```